rules/no-zeebe-properties.js

```diff
@@ -8,10 +6,6 @@ const { skipInNonExecutableProcess } = require('./utils/rule');
 
 module.exports = skipInNonExecutableProcess(function() {
   function check(node, reporter) {
-    if (!is(node, 'zeebe:PropertiesHolder')) {
-      return;
-    }
-
```

Comment:
We removed this type from the Zeebe model extension (cf. https://github.com/camunda/zeebe-bpmn-moddle/pull/44 and https://github.com/bpmn-io/bpmn-js-properties-panel/pull/861) because any element can have these extension properties. So there's no need for `zeebe:PropertiesHolder`. Same in Camunda 7.